examples/package-lock.json

```diff
@@ -54,6 +54,7 @@
         "@opentelemetry/instrumentation-redis-4": "^0.42.0",
         "@opentelemetry/instrumentation-restify": "^0.41.0",
         "@opentelemetry/instrumentation-router": "^0.40.0",
+        "@opentelemetry/instrumentation-runtime-node": "file:../../../../otel/opentelemetry-js-contrib/plugins/node/instrumentation-runtime-node",
```

Comment:
This definitely seems wrong. Probably from local dev work?

Reply:
same as `"dev": true`. I'll run `npm i` again and, if not fixed, I'll change it manually.

Thanks for catching it :)